packages/brokeneck-react/src/components/Navigation.js

```diff
@@ -6,49 +6,95 @@ import {
   IconButton,
   makeStyles,
   Tab,
-  Tabs,
-  Typography
+  Tabs
 } from '@material-ui/core'
+import { useTheme } from '@material-ui/core/styles'
+import GroupIcon from '@material-ui/icons/Group'
+import UserIcon from '@material-ui/icons/Person'
+
+import MoonIcon from '../icons/moon'
+import SunIcon from '../icons/sun'
 
 import { ThemeSwitcherContext } from './ThemeSwitcherProvider'
 import Provider from './Provider'
 
 const useStyles = makeStyles(theme => ({
   appBar: {
+    alignItems: 'center',
+    backgroundColor: theme.palette.headerBackground.main,
+    color: theme.palette.bodyText.main,
+    flexDirection: 'row',
     marginBottom: theme.spacing(3),
-    flexDirection: 'row'
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(4)
+  },
+  provider: {
+    marginLeft: 'auto'
   },
-  actions: {
-    marginLeft: 'auto',
-    '& > * + *': {
-      marginLeft: theme.spacing(1)
-    }
+  switcher: {
+    borderLeft: `1px solid ${theme.palette.separator.main}`,
+    borderRight: `1px solid transparent`, // Needed for symetry
+    height: '42px',
+    marginLeft: theme.spacing(2),
+    paddingLeft: theme.spacing(2),
+    width: '42px'
+  },
+  tabRoot: {
+    minWidth: 120
+  },
+  tabWrapper: {
+    flexDirection: 'row'
   }
 }))
 
 export default function Navigation() {
   const classes = useStyles()
+  const {
+    palette: { type: themeType }
+  } = useTheme()
   const toggleThemeType = useContext(ThemeSwitcherContext)
   const isGroups = useRouteMatch('/groups')
 
   const currentTab = isGroups ? 'groups' : 'users'
 
   return (
-    <AppBar position="static" color="primary" className={classes.appBar}>
+    <AppBar position="static" className={classes.appBar}>
       <Tabs value={currentTab}>
-        <Tab component={NavLink} to="/users" label="Users" value="users" />
-        <Tab component={NavLink} to="/groups" label="Groups" value="groups" />
+        <Tab
+          classes={{ root: classes.tabRoot, wrapper: classes.tabWrapper }}
+          component={NavLink}
+          to="/users"
+          label={
+            <>
+              <UserIcon style={{ verticalAlign: 'middle' }} /> Users
```

Comment:
let's avoid inline styles and use classes consistently

Reply:
Good catch, that was a leftover from some of my tests. This is covered by the `display: flex` on the Tab anyway, so I've removed it.